webapp/components/test-run.html

```diff
@@ -58,14 +58,18 @@
         </template>
       </template>
 
-      <template is="dom-if" if="{{ !isDiff(testRun.browser_name) }}">
-        <paper-tooltip offset=0>
+      <paper-tooltip offset=0>
+        <template is="dom-if" if="{{ !isDiff(testRun.browser_name) }}">
           {{displayName(testRun.browser_name)}} {{testRun.browser_version}}<br>
           Labels: {{displayLabels(testRun.labels)}}<br>
           Started {{timeFormat(testRun.time_start)}} {{timeTaken(testRun)}}<br>
           {{moreTooltip(testRun)}}
-        </paper-tooltip>
-      </template>
+        </template>
+        <template is="dom-if" if="{{ isDiff(testRun.browser_name }}">
```

Comment:
This tooltip shows up as part of the browser tooltips too, which I think isn't intended. I don't even see how it's possible given that the conditions are negated. Found https://github.com/Polymer/polymer/issues/2165 while looking for a construct that would make the bug impossible.

Reply:
Ah; missing a trailing `)`